recipes/nr.cli/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "nr.cli" %}
+{% set version = "1.2.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "1bf47100b8ca21b3f184ee6f27c34adcfb7007612f8f8aada570891ffed59bca"
+
+build:
+  number: 0
+  entry_points:
+    - nr = nr.cli:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
```

Comment:
remove setuptools?

Reply:
Same here, can it actually be removed from the host section if the setup script imports `setuptools`?